modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -245,6 +245,13 @@ TEST_P(Test_ONNX_layers, Reshape)
     testONNXModels("unsqueeze");
 }
 
+TEST_P(Test_ONNX_layers, Slice)
+{
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE)
+        throw SkipTestException("Only OCV backend is supported");
```

Comment:
`Slice` layer has been updated recently (https://github.com/opencv/opencv/pull/14626). Can you check IE backend as well?